pkgs/servers/mail/mailman/default.nix

```diff
@@ -1,35 +1,32 @@
-{ lib, python3, fetchPypi, fetchpatch, pythonOlder, postfix, lynx
+{ lib, buildPythonPackage, fetchPypi, fetchpatch, pythonOlder, python3, postfix, lynx
 }:
 
 let
-  py = python3.override {
+  # Mailman does not support sqlalchemy >= 1.4 https://gitlab.com/mailman/mailman/-/issues/845
+  pythonOverride = python3.override {
     packageOverrides = self: super: {
-      # https://gitlab.com/mailman/mailman/-/issues/845
       sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
         version = "1.3.24";
         src = oldAttrs.src.override {
           inherit version;
-          sha256 = "06bmxzssc66cblk1hamskyv5q3xf1nh1py3vi6dka4lkpxy7gfzb";
+          sha256 = "ebbb777cbf9312359b897bf81ba00dae0f5cb69fba2a18265dcc18a6f5ef7519";
```

Comment:
Why change the hash when we're not changing the source?

Reply:
i updated mailman in this pr before rebasing and left those in